pages/common/gnatprep.md

```diff
@@ -0,0 +1,12 @@
+# gnatprep
+
+> Preprocessor for Ada source code files (part of the GNAT toolchain).
+> More information: <https://gcc.gnu.org/onlinedocs/gnat_ugn/Preprocessing-with-gnatprep.html>.
+
+- Use symbol definitions from a file:
```

Comment:
use for what?

Reply:
For looking up values associated with symbol names and substituting those names in source code files.

I don't really understand how you want this wording improved. Are you implying it should be understandable for people who don't know what a preprocessor does, or your point is that it should be like "**Load** symbol definitions from a file"?